plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckPulsingAnimator.kt

```diff
@@ -1,12 +1,15 @@
 package com.mapbox.maps.plugin.locationcomponent.animators
 
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
 import android.graphics.Color
 import androidx.annotation.ColorInt
 import androidx.core.view.animation.PathInterpolatorCompat
 
-internal class PuckPulsingAnimator : PuckAnimator<Double>(Evaluators.DOUBLE) {
+internal class PuckPulsingAnimator(private val pixelRatio: Float = 1.0f) :
+  PuckAnimator<Double>(Evaluators.DOUBLE) {
+  var maxRadius: Double = DEFAULT_RADIUS * pixelRatio
```

Comment:
it's actually kinda breaking change isn't it? previously it was always 10.0 while now it will be more (as pixel ratio is more than 1.0 on absolute majority of devices).

Reply:
Actually, we already use `default_value * pixelratio` when applying the default settings of LocationComponent (see : https://github.com/mapbox/mapbox-maps-android/blob/main/plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/generated/LocationComponentAttributeParser.kt#L30). I have sync'd this file to use pixelRatio too (only for default value). I have tested and the default behavior is the same. 

Although this is true only for default value, for user provided value (we don't use pixelRatio, somehow🤔) I think  we should also use pixelRatio for user-defined `pulsingMaxRadius` values too, but that would be behavioural change.